src/dataLayer/mongo/user.js

```diff
@@ -58,11 +58,6 @@ export async function createUser(root, vars, ctx) {
       'Something went wrong creating your account, please try again'
     );
   } else {
-    console.error(
-      'Cannot create account, ' +
-        'the unique id associated with this user is already in use',
-      accountLinkId
-    );
-    return exists[0];
+    throw new Error('The accountLinkId is already in use');
```

Comment:
Could we please change this to "Account already in use"? -I'd like to keep accountLinkId internal.